### DISCUSSION TITLE
What is CDN and how does it work?سی‌دی‌ان چیه و چطور کار می‌کنه؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#114] [Q&A سوالات رایج]
سی‌دی‌ان چیه و چطور کار میکنه و تو پنل چجوری تنظیمش کنم؟

**Comment by shuvvan** [comment]
؟؟؟؟؟کسی ظاهرا پاسخ نمیده

**Comment by zZedix** [comment]
واژه CDN مخفف Content Delivery Network هست

داخل این روش ارتباطی ارائه دهنده CDN میاد برقراری ارتباط با سرور اصلی پشت یک دامنه رو از مسیر های نزدیک تر میبره که دسترسی به سایت یا اطلاعات سریع تر صورت بگیره .
به عنوان مثال شما میخوای به سایت google متصل بشی که سرور مرکزیش آمریکاست ! اگر این ارتباط  قرار باشه مستقیم صورت بگیره برای کاربر هایی که از آمریکا دور هستند عملا این ارتباط خیلی کند میشه ! با استفاده از CDN شما میای مثلا به سرور ترکیه وصل میشی بعد از اون به اروپا و در آخر به آمریکا !
بزرگترین حسن استفاده از CDN پنهان موندن IP سرور اصلی شما ( Origin Server ) از چشم سیستم فیلترینگ
